### ISSUE TITLE
Crash when pasting content

### ISSUE THREAD

**Issue by Reinmar** [ckeditor/github-writer#243]
Steps:

1. Type "Foo @Reinmar"
2. Cut that.
3. Paste that.

```
Uncaught TypeError: Converting circular structure to JSON
    --> starting at object with constructor 'Object'
    |     property 'writer' -> object with constructor 'dc'
    |     property 'model' -> object with constructor 'Gc'
    |     ...
    |     property 'attributes' -> object with constructor 'Object'
    --- property 'mention' closes the circle
    at JSON.stringify (<anonymous>)
    at Gc.e (github-writer.js:1708)
    at Gc.fire (github-writer.js:59)
    at xc.<anonymous> (github-writer.js:1678)
    at xc.fire (github-writer.js:59)
    at xc._handleChangeBlock (github-writer.js:505)
    at Gc._runPendingChanges (github-writer.js:548)
    at Gc.change (github-writer.js:543)
    at pd.listenTo.priority (github-writer.js:785)
    at pd.fire (github-writer.js:59)
```

**Comment by fredck** [comment]
This one was weird to debug but I have the impression that it is related to a CKEditor issue which I've described here: ckeditor/ckeditor5#8792.

@Reinmar, can you confirm (in the issue itself) that the above CKEditor issue is valid and if it is a recent regression?

**Comment by Reinmar** [comment]
https://github.com/ckeditor/ckeditor5/issues/8792 is already fixed on `master`, so you could try to check whether it fixes the issue in GH Writer.

**Comment by fredck** [comment]
This is confirmed to be fixed following the upgrade to CKEditor 5 v25.0.0 with 4feedde.
